.changeset/brown-timers-hang.md

```diff
@@ -0,0 +1,20 @@
+---
+'@vanilla-extract/babel-plugin-debug-ids': patch
+'@vanilla-extract/css-utils': patch
+'@vanilla-extract/css': patch
+'@vanilla-extract/dynamic': patch
```

Comment:
The overridden browser packages in the babel config (e.g. css, dynamic, etc) aren't affected by this right? Don't think those ones need a release.